.github/workflows/irap-fedora-ci-runner.yaml

```diff
@@ -0,0 +1,49 @@
+name: Build and Push atlas-irap-env container
+
+on:
+  push:
+    branches: 
+      - master
+    paths:
+      - "irap-fedora/**"
+  pull_request:
+    paths:
+      - "irap-fedora/**"
+
+jobs:          
+  deploy:
+    name: Build and deploy image to quay.io/ebigxa
+    runs-on: ubuntu-20.04
+    env:
+      IMAGE_NAME: irap_fedora
```

Comment:
The IMAGE_NAME should come as well from the tag file, as the tag comes. I'm sure the CI for conda templates has this already and you should be able to copy paste.

Reply:
Done in 69f474a98323110c9e6958f04634890e27de9c2b